core/src/types.rs

```diff
@@ -689,6 +689,8 @@ const INVALID_PROJECT_NAME: &str = r#"
 Project name must only contain alphanumeric characters.
 "#;
 
+pub const MULTI_PROOF_CELL_DIMS: (u16, u16) = (16, 64);
```

Comment:
Maybe to write a function which returns hardcoded dimenstions instead of having constant pair, which is never used as pair?

like:

```rust
pub fn multi_proof_dimensions() -> Dimensions {
    Dimensions::new(16, 64).expect("...")
}
```